<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>663</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTextEdit" name="textEdit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/icon/images/morty.png);</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>357</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="actionNew_N"/>
    <addaction name="actionOpen_O"/>
    <addaction name="actionSave_S"/>
    <addaction name="actionSave_as_A"/>
    <addaction name="separator"/>
    <addaction name="actionPrint_P"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="actionUndo_U"/>
    <addaction name="actionRedo_R"/>
    <addaction name="separator"/>
    <addaction name="actionCopy_C"/>
    <addaction name="actionCut_C"/>
    <addaction name="actionPaste_P"/>
    <addaction name="actionSelectAll_A"/>
    <addaction name="separator"/>
    <addaction name="actionDate_D"/>
    <addaction name="actionFont_F"/>
    <addaction name="actionColor_C"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="actionUpdate_U"/>
    <addaction name="actionAbout_Soft_S"/>
    <addaction name="action1"/>
    <addaction name="aboutWebAction"/>
    <addaction name="aboutQtAction"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_N"/>
   <addaction name="actionOpen_O"/>
   <addaction name="actionSave_S"/>
   <addaction name="separator"/>
   <addaction name="actionCopy_C"/>
   <addaction name="actionPaste_P"/>
   <addaction name="actionCut_C"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew_N">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/icon/images/New Icon.jpg</normaloff>:/new/icon/images/New Icon.jpg</iconset>
   </property>
   <property name="text">
    <string>New(&amp;N)</string>
   </property>
  </action>
  <action name="actionOpen_O">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/icon/images/Open Icon.jpg</normaloff>:/new/icon/images/Open Icon.jpg</iconset>
   </property>
   <property name="text">
    <string>Open(&amp;O)</string>
   </property>
  </action>
  <action name="actionSave_S">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/icon/images/Save Icon.jpg</normaloff>:/new/icon/images/Save Icon.jpg</iconset>
   </property>
   <property name="text">
    <string>Save(&amp;S)</string>
   </property>
  </action>
  <action name="actionSave_as_A">
   <property name="text">
    <string>Save as(&amp;A)</string>
   </property>
  </action>
  <action name="actionPrint_P">
   <property name="text">
    <string>Print(&amp;P)</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit()</string>
   </property>
  </action>
  <action name="actionUndo_U">
   <property name="text">
    <string>Undo(&amp;U)</string>
   </property>
  </action>
  <action name="actionRedo_R">
   <property name="text">
    <string>Redo(&amp;R)</string>
   </property>
  </action>
  <action name="actionCopy_C">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/icon/images/Copy Icon.jpg</normaloff>:/new/icon/images/Copy Icon.jpg</iconset>
   </property>
   <property name="text">
    <string>Copy(&amp;C)</string>
   </property>
  </action>
  <action name="actionPaste_P">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/icon/images/Paste.png</normaloff>:/new/icon/images/Paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste(&amp;P)</string>
   </property>
  </action>
  <action name="actionCut_C">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/icon/images/Cut Icon.jpg</normaloff>:/new/icon/images/Cut Icon.jpg</iconset>
   </property>
   <property name="text">
    <string>Cut(&amp;C)</string>
   </property>
  </action>
  <action name="actionSelectAll_A">
   <property name="text">
    <string>Select All(&amp;S)</string>
   </property>
  </action>
  <action name="actionDate_D">
   <property name="text">
    <string>Date(&amp;D)</string>
   </property>
  </action>
  <action name="actionFont_F">
   <property name="text">
    <string>Font(&amp;F)</string>
   </property>
  </action>
  <action name="actionColor_C">
   <property name="text">
    <string>Color(&amp;C)</string>
   </property>
  </action>
  <action name="actionUpdate_U">
   <property name="text">
    <string>更新(&amp;U)</string>
   </property>
  </action>
  <action name="actionAbout_Soft_S">
   <property name="text">
    <string>关于软件(&amp;S)</string>
   </property>
  </action>
  <action name="action1">
   <property name="text">
    <string>帮助(&amp;E)</string>
   </property>
  </action>
  <action name="aboutWebAction">
   <property name="text">
    <string>关于官网(&amp;W)</string>
   </property>
  </action>
  <action name="aboutQtAction">
   <property name="text">
    <string>关于Qt(&amp;Q)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
